languages/go/internal/host/host.go

```diff
@@ -388,10 +390,14 @@ func (h Host) ToGo(v types.Term) (interface{}, error) {
 	case ValueVariable:
 		return inner, nil
 	case ValueExpression:
-		return nil, fmt.Errorf(
-			"Received Expression from Polar VM. The Expression type is not yet supported in this language.\n" +
-				"This may mean you performed an operation in your policy over an unbound variable.")
+		if !h.acceptExpression {
+			return nil, &errors.UnexpectedExpressionError{}
+		}
```

Comment:
Just realised this needs a `return inner, nil`. There's not much you can do with the expression at the moment, but this would give someone a chance!